### ISSUE TITLE
Windows 10 move files: NVDA cannot tab to buttons in the file replace dialog

### ISSUE THREAD

**Issue by PratikP1** [nvaccess/nvda#6951]
In Windows 10 Explorer, when moving files from one folder to another and duplicates are found, NVDA remains focused on the first button, which is "replace." Depending on the context, two additional buttons appear in this dialog. These could be one of "skip all," "skip," "compare files," or "compare each."

* Actual behavior: NVDA can use object navigation to move to each button presented and activate it. 
* Expected behavior: NVDA should be able to tab to each button and use standard navigation and activation commands such as tab and space for these functions.

I can provide steps to reproduce but they'll be specific to my system.

Test configuration: Windows 10 Build 15048
NVDA version next-13899,af44c1d4

**Comment by josephsl** [comment]
Hi, Does Narrator exhibit this problem too? Thanks.

**Comment by k-kolev1985** [comment]
As far as I know, this is by design from Microsoft. The 3 buttons in question are placed one below the other. You don't use TAB or Shift+TAB to navigate between them - you use the UP and DOWN arrow keys to do that.

**Comment by PratikP1** [comment]
Both Narrator and NVDA behaves the same. Thank you for pointing out that Microsoft has changed the paradigm and now requires up/down arrow keys.

**Comment by jcsteh** [comment]
Closing, as this is not an NVDA issue. If you believe this interaction to be incorrect, please provide feedback to Microsoft. Thanks.
